events.json

```diff
@@ -1,291 +1,34 @@
 [
```

Comment:
We shouldn't be committing anything to the JSON because it might change. This time I will accept it as its only cleaning it